app.js

```diff
@@ -8,15 +8,19 @@ const {
   getSpecificReview,
   getReviews,
   getReviewComments,
+  updateReviewVotes,
   postReviewComment,
+  getUsers
 } = require('./controller');
 
 app.get('/api', standardResponse);
 app.get('/api/categories', getCategories);
 app.get('/api/reviews', getReviews);
 app.get('/api/:review_Id/comments', getReviewComments);
 app.get('/api/review/:review_Id', getSpecificReview);
+app.patch('/api/review/:review_Id', updateReviewVotes);
 app.post('/api/review/:review_Id/comments', postReviewComment);
+app.get('/api/users', getUsers);
 
```

Comment:
as a refactor suggestion, have a think about what level of bloat you're willing to accept, and at what point you start to organise/categorise your controllers and models into their own files